order.go

```diff
@@ -46,17 +46,41 @@ type OrderUpdateParams struct {
 	Status                 OrderStatus
 }
 
+// OrderReturnParams is the set of parameters that can be used when returning
+// orders. For more details, see: https://stripe.com/docs/api#return_order.
+type OrderReturnParams struct {
+	Params
+	Items []*OrderItemParams
```

Comment:
They're referring to the same resource in the API so I think it's extremely unlikely that they'll ever diverge. Basically returns specify a subset of the items that the order were created with. The only difference is that you can get away with specifying a subset of the `OrderItemParams` on returns because we can infer the rest, but they fundamentally refer to the same data. Internally we also use the same definition for these so I think it's peaceful leaving as is (: